content/en/developers/marketplace/_index.md

```diff
@@ -63,6 +63,10 @@ There are a few different types of offerings that you can list on the Datadog In
 3. SaaS license or subscription (Marketplace only)
 4. Professional services (Marketplace only)
 
+Regardless of the type of offering that you are developing, you will build out a tile on the Marktplace or Integrations page that represents your listing. You can see an example of a Marketplace tile below:
```

Comment:
```suggestion
Create an image to use for the tile on the Marketplace or Integrations page that represents your listing. You can see an example of a Marketplace tile below:
```

Reply:
Per discussion in Slack, you can keep the original language, but please remove the "will" so the sentence is in present tense.

Reply:
Thanks @urseberry , updated in latest commit